src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -229,9 +229,11 @@ def generate_reports(self, report: Report) -> None:
             logger.critical("The main repository analysis failed. Cannot generate a report for it.")
             return
 
-        output_target_path = os.path.join(
-            global_config.output_path, "reports", git_url.get_repo_dir_name(report.root_record.context.remote_path)
-        )
+        target_dir = git_url.get_repo_dir_name(report.root_record.context.remote_path)
+        if not target_dir:
+            target_dir = report.root_record.context.repo_full_name
```

Comment:
```suggestion
        target_dir = git_url.get_repo_dir_name(report.root_record.context.remote_path) or report.root_record.context.repo_full_name
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/153/commits/bdeba388db589ff62f060dbb87c5512bb05ba587